src/display/canvas.js

```diff
@@ -956,6 +962,11 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
             break;
         }
       }
+
+      if (this.blendModeFallback) {
+        this.ctx.globalAlpha = 0.5;
```

Comment:
Since we aren't supporting the actual blend mode here, 50% was chosen to allow equal visibility of the top and bottom content

Reply:
Please add that as a comment here.

Reply:
added